ui/App.svelte

```diff
@@ -25,23 +25,23 @@
   import Project from "./Screen/Project.svelte";
   import ProjectCreation from "./Screen/ProjectCreation.svelte";
   import ProjectRegistration from "./Screen/ProjectRegistration.svelte";
-  import Search from "./Screen/Search.svelte";
   import SendFunds from "./Screen/SendFunds.svelte";
   import Settings from "./Screen/Settings.svelte";
   import TransactionDetails from "./Screen/TransactionDetails.svelte";
+  import Untracked from "./Screen/Project/Untracked.svelte";
```

Comment:
I agree that `Untracked` should be a state of Project (see note in PR description), but I would prefer to do the `Project` screen refactor in a new PR because it'll touch a lot of different things (issue for that [here](https://github.com/radicle-dev/radicle-upstream/issues/737)).

Reply:
Sounds good!